app/data.py

```diff
@@ -0,0 +1,33 @@
+from typing import IO
+
+from app.clien import Client
+from app.shops import Shop
+
+
+def create_clients(data: IO) -> dict:
+    clients = {}
+
+    for client_data in data["customers"]:
+        name = client_data["name"]
+        client = Client(
+            name=name,
+            product_cart=client_data["product_cart"],
+            location=client_data["location"],
+            money=client_data["money"],
+            car=client_data["car"]
+        )
+        clients[name] = client
+    return clients
```

Comment:
Use list comprehension and unpacking (using `**`)
Hint: You can place Car creation inside Customer init